cmd/test/main.go

```diff
@@ -472,7 +477,25 @@ func test(ctx *TestCommandContext, paths []string, testCommandData *TestCommandD
 		CliVersion:            ctx.CliVersion,
 		IsCI:                  ctx.CiContext.IsCI,
 		Quiet:                 testCommandData.Quiet,
-	})
+	}
+	err = evaluation.PrintResults(evaluationData)
+
+	if testCommandData.SaveResults != "" {
+		if evaluation.IsWritableDirectory(testCommandData.SaveResults) {
+			resultsText, err := evaluation.GetjsonResult(evaluationData)
+			if err != nil {
+				return err
+			}
+
+			err = ioutil.WriteFile(testCommandData.SaveResults, []byte(resultsText), 0666)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
This print seems a bit redundant as well, since the error returned will be printed out